### PR TITLE
Add installation for shared hosting

### PR DESCRIPTION
**Pull request by metallurgical**

### CODE REVIEW

**Comment by hanisirfan** [comment]
Can you please move this text to the Shared Hosting section?
image

File: docs/installation.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ There's **2** methods that you can use to install the system.
 - Run `npm run prod` to and compile the assests.
 - Generate the application key with `php artisan key:generate` command. This key will be stored in `.env` file and be used for encryption.
 - (Optional) For production environment, set the `APP_DEBUG` value to `false` to prevent sensitive config to be exposed to the user.
-- Upload the files to your web server.
+- Upload the files to your web server. For installation in shared hosting, upload together with `vendor` folder. Head over to `Shared Hosting` section for further configurations.
```
Comment (hanisirfan) [inline docs/installation.md]: Change `Shared Hosting` to [Shared Hosting](#shared-hosting).

**Review by hanisirfan — CHANGES_REQUESTED** [review]

**Comment by metallurgical** [comment]
Changes updated

File: docs/installation.md (hunk 1 above)
Comment (metallurgical) [reply docs/installation.md]: sure

**Review by hanisirfan — APPROVED** [review]

**Comment by hanisirfan** [comment]
Nice!

**Pull request merged**